### DISCUSSION TITLE
Add filters to "Create new delivery" screen

### DISCUSSION THREAD

**Discussion by Ian-Highlander** [gameserverapp/Platform#877] [Suggestions]
Just a QOL thing really, when using the "Create new delivery" screen to send something to players in game from our shop packs it's getting harder and harder to find the right one in the list.

We have hundreds of items on both our ASE and ASA cluster in-game shops and just scrolling and trying to find the right one (let alone picking the ASE one by mistake or vice-versa when trying to order as they're named the same) is getting more and more cumbersome.

Is there any way we can get the filters added to this screen to allow us to filter the drop down list we're selecting from at all please?

![image](https://github.com/gameserverapp/Platform/assets/109648228/0f679f2f-1604-4b4f-92cc-2e775257adc6)

Even some of the filters that are in the shop packs screen available in this screen would be a massive improvement.

![image](https://github.com/gameserverapp/Platform/assets/109648228/4a718108-444f-477e-9f8e-fd9562250aa1)

**Comment by mvaessen** [comment]
Thanks for your suggestion. Will continue updating in #940
